libsolidity/parsing/Parser.cpp

```diff
@@ -811,6 +811,14 @@ ASTPointer<VariableDeclaration> Parser::parseVariableDeclaration(
 					case Token::CallData:
 						location = VariableDeclaration::Location::CallData;
 						break;
+					case Token::Code:
+						location = VariableDeclaration::Location::Code;
+						syntaxError(
```

Comment:
I think an `solUnimplementedError` (or something like that, not sure what the exact name is) would be desired here